gallery/gallery-war/src/main/webapp/WEB-INF/tags/silverpeas/gallery/editMedia.tag

```diff
@@ -171,6 +172,7 @@
       </c:if>
 
       var dateErrors;
+      let errorPromise = sp.promise.resolveDirectlyWith();
```

Comment:
Be cautious: this var is overriden by your code if the if-statement is validated.
In order to keep the idea behind, the above code should be replaced by this one:
```javascript
let errorPromises = [sp.promise.resolveDirectlyWith()];
```
So, your promise can be added into the array. See below my next comment

Reply:
After seeing with Yohann who has proposed you the solution, the better way is to initialize an empty array:
```javascript
let errorPromises = [];
```

Reply:
Done.